### ISSUE TITLE
On 5.2.1, getOrders doesn't work

### ISSUE THREAD

**Issue by Senpai10** [jlevers/selling-partner-api#352]
Hello,

First of all, thank you for your lib !

I updated the package from 5.1.1 to 5.2.1 version. When I try to call getOrders, I get:

```
Exception when calling OrdersV0Api->getOrders: [400] {
  "errors": [
    {
      "code": "InvalidInput",
      "message": "Application do not have access to some or all requested resource",
      "details": ""
    }
  ]
}

```
So I downgraded to 5.1.1 to check it, because I never had this error before. 

Result: getOrders works.

Did something changed on 5.2.1 ?

**Comment by 0xSekar** [comment]
@Senpai10 , see issue #350 . Version 5.2.1 is trying to sign your order request. If your app don't have the relevant roles, the RDT request fails not allowing to retrieve orders

**Comment by Senpai10** [comment]
@0xSekar Thank you for your answer.

So from now, we have to call GetOrders with the RDT token ?
Or is it possible to make the GetOrders call without RDT token ? I’m not looking for the PII information but only orders informations.

**Comment by 0xSekar** [comment]
Yes, at least until the issue is fixed you need to have RDT enabled, or downgrade to 5.1.1 until the fix is released

**Comment by jlevers** [comment]
This should be fixed in `v5.2.3`, give it a try and let me know if you run into any more issues!

**Comment by Senpai10** [comment]
Thank you ! It's fixed from my side
